proxmoxtf/resource/firewall/rules.go

```diff
@@ -187,7 +188,7 @@ func Rules() *schema.Resource {
 }
 
 // RulesImport imports firewall rules.
-func RulesImport(ctx context.Context, d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
+func RulesImport(ctx context.Context, d *schema.ResourceData, m any) ([]*schema.ResourceData, error) {
 	id := d.Id()
```

Comment:
I'm totally in favour of cleaning up and modernizing the old code to follow new conventions, but could you possibly extract these changes into a separate PR next time? It would help reduce the cognitive load for reviewers 😅.